Gopkg.toml

```diff
@@ -91,19 +85,14 @@ required = [
   name = "github.com/stretchr/testify"
   version = "1.3.0"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/wadey/gocovmerge"
-
 
 [[override]]
   name = "github.com/golang/protobuf"
   version = "1.3.1"
 
 [[constraint]]
   name = "github.com/redhat-developer/devconsole-api"
-  packages = [
-    "pkg/apis",
-    "pkg/apis/devconsole/v1alpha1",
-  ]
+  source = "github.com/corinnekrych/devconsole-api"
```

Comment:
Should probably point to redhat-developer/devconsole-api instead of corinnekrych/devconsole-api?

Reply:
yes. this is temporary. will remove it once https://github.com/redhat-developer/devconsole-api/pull/23 is merged